tests/test_optimizer_ax.py

```diff
@@ -0,0 +1,315 @@
+try:
+    from ax import RangeParameter, ParameterType
+    ax_imported = True
+except ImportError:
+    ax_imported = False
+
+import numpy as np
+import sys
+import unittest
+
+from CADETProcess.optimization import OptimizationProblem
+from CADETProcess.comparison import Comparator
+from CADETProcess.optimization.axAdapater import AxInterface
```

Comment:
Use concrete implementations of AxInterface. Include different tests. Also check in behaviour tests!

Reply:
This script is from the beginning of my work with Ax. I think we could use parts of it for system integration tests, but maybe the whole file is not necessary any longer. Let's discuss this.